exammode/models.py

```diff
@@ -0,0 +1,120 @@
+from django.db import models
+from django.utils import timezone
+
+from course.models import CourseInstance, CourseModule
+from exercise.exercise_models import LearningObject
+from userprofile.models import UserProfile
+
+# Create your models here.
+
+
+class ExamSessionManager(models.Manager):
```

Comment:
`ExamSessionManager` could be named `ActiveExamSessionManager` since it is restricted to active sessions.

Reply:
Fixed